### PR TITLE
Initialize Common Label for ID Localization.

### PR DESCRIPTION
**Pull request by irvifa**
Address https://github.com/kubernetes/website/issues/15341

### CODE REVIEW

**Comment by irvifa** [comment]
Related to https://github.com/jk8s/sig-docs-id/issues/9

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+title: Label yang Disarankan
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+Kamu dapat melakukan visualisasi dan mengatur objek Kubernetes dengan lebih banyak _tools_ 
+dibandingkan dengan perintah kubectl dan dasbor. Sekumpulans et label mengizinkan _tools_
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]: typo
```suggestion
dibandingkan dengan perintah kubectl dan dasbor. Sekumpulan label mengizinkan _tools_ 
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
+---
+title: Label yang Disarankan
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+Kamu dapat melakukan visualisasi dan mengatur objek Kubernetes dengan lebih banyak _tools_ 
+dibandingkan dengan perintah kubectl dan dasbor. Sekumpulans et label mengizinkan _tools_ 
+untuk bekerja secara _interoperably_, mendeskripsikan objek dengan cara yang umum yang dapat
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]: according to glosarium, we can use `interoperabilitas`
```suggestion
untuk bekerja dengan interoperabilitas, mendeskripsikan objek dengan cara yang umum yang dapat 
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+Kamu dapat melakukan visualisasi dan mengatur objek Kubernetes dengan lebih banyak _tools_ 
+dibandingkan dengan perintah kubectl dan dasbor. Sekumpulans et label mengizinkan _tools_ 
+untuk bekerja secara _interoperably_, mendeskripsikan objek dengan cara yang umum yang dapat 
+dipahami semua _tools_.
+
+Sebagai tambahaknbagi _tooling_ tambahan, label yang disarankan ini mendeskripsikan
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]: typo
```suggestion
Sebagai tambahan bagi _tooling_ tambahan, label yang disarankan ini mendeskripsikan 
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+dibandingkan dengan perintah kubectl dan dasbor. Sekumpulans et label mengizinkan _tools_ 
+untuk bekerja secara _interoperably_, mendeskripsikan objek dengan cara yang umum yang dapat 
+dipahami semua _tools_.
+
+Sebagai tambahaknbagi _tooling_ tambahan, label yang disarankan ini mendeskripsikan 
+aplikasi sehingga informasi yang ada diapat di-_query_.
+{{% /capture %}}
+
+{{% capture body %}}
+Metadata ini diorganisasi ebrbasis konsep dari sebuah  _application_. Kubernetes bukan merupakan
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]: typo
```suggestion
Metadata ini diorganisasi berbasis konsep dari sebuah  aplikasi. Kubernetes bukan merupakan 
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+untuk bekerja secara _interoperably_, mendeskripsikan objek dengan cara yang umum yang dapat 
+dipahami semua _tools_.
+
+Sebagai tambahaknbagi _tooling_ tambahan, label yang disarankan ini mendeskripsikan 
+aplikasi sehingga informasi yang ada diapat di-_query_.
+{{% /capture %}}
+
+{{% capture body %}}
+Metadata ini diorganisasi ebrbasis konsep dari sebuah  _application_. Kubernetes bukan merupakan 
+sebuah platfor sebagai sebuah service (_platform as a service_/PaaS) dan tidak
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]: typo
```suggestion
sebuah platform sebagai sebuah _service_ (_platform as a service_/PaaS) dan tidak 
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+Sebagai gantinya, aplikasi merupakan suatu hal informal yang dideskripsikan melalui metadata. 
+Definisi yang dimiliki oleh sebuah aplikasi merupakan sebuah hal yang cukup longgar.
+
+{{< note >}}
+Berikut merupakan label yang disarankan. Label ini mempermudah 
+proses manajemen aplikasi tetapi tidak dibutuhkan untuk _tooling_ utama apa pun.
+{{< /note >}}
+
+Label yang digunakan secara umum serta anotasi memiliki prefiks yang serupa: `app.kubernetes.io`. Label
+tanpa sebuah prefiks bersifat privat terdapat pengguna saja. Prefiks yang digunakan secara umum tadi
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
tanpa sebuah prefiks bersifat privat khusus pengguna saja. Prefiks yang digunakan secara umum tadi
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+menjamin bahwa label tadi tidak akan mengganggu label kustom yang diberikan oleh pengguna.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]: kustom is not indonesian word
```suggestion
menjamin bahwa label tadi tidak akan mengganggu label _custom_ yang diberikan oleh pengguna.
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+## Label
+
+Untuk mendapatkan keuntungan menyeluruh dari penggunaan label ini, 
+label harus digunakan pada seluruh objek sumber daya.
+
+| _Key_                               | Deskripsi             | Contoh   | Tipe |
+| ----------------------------------- | --------------------- | -------- | ---- |
+| `app.kubernetes.io/name`            | Nama aplikasi | `mysql` | string |
+| `app.kubernetes.io/instance`        | Nama unik yang bersifat sebagai identifier dari sebuah instans aplikasi | `wordpress-abcxzy` | string |
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
| `app.kubernetes.io/instance`        | Nama unik yang bersifat sebagai pengidentifikasi dari sebuah instans aplikasi | `wordpress-abcxzy` | string |
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+| `app.kubernetes.io/version`         | Versi saat ini dari aplikasi (misalnya sebuah version semantik, hash revisi, etc.) | `5.7.21` | string |
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
| `app.kubernetes.io/version`         | Versi saat ini dari aplikasi (misalnya sebuah versi semantik, hash revisi, etc.) | `5.7.21` | string |
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+| `app.kubernetes.io/component`       | Komponen yang ada pada arsitektur | `database` | string |
+| `app.kubernetes.io/part-of`         | Nama dari komponen lebih tinggi dari aplikasi yang mencakup bagian ini | `wordpress` | string |
+| `app.kubernetes.io/managed-by`  | Alat yang digunakan untuk mengatur operasi pada aplikasi | `helm` | string |
+
+Untuk memberikan ilustrasu dari penggunaan label, bayangkan sebuah objek StatefulSet yang didefinisikan sebagai berikut:
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]: typo
```suggestion
Untuk memberikan ilustrasi dari penggunaan label, bayangkan sebuah objek StatefulSet yang didefinisikan sebagai berikut:
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+    app.kubernetes.io/instance: wordpress-abcxzy
+    app.kubernetes.io/version: "5.7.21"
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: wordpress
+    app.kubernetes.io/managed-by: helm
+```
+
+## Aplikasi dan Instans Aplikasi
+
+Sebuah aplikasi dapat diinstal satu atau lebih di dalam kluster Kubernetes dan,
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
Sebuah aplikasi dapat diinstal sekali atau beberapa kali di dalam kluster Kubernetes dan, 
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 12]
```diff
@@ -0,0 +1,179 @@
...
+    app.kubernetes.io/version: "5.7.21"
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: wordpress
+    app.kubernetes.io/managed-by: helm
+```
+
+## Aplikasi dan Instans Aplikasi
+
+Sebuah aplikasi dapat diinstal satu atau lebih di dalam kluster Kubernetes dan, 
+pada beberapa kasus, di dalam sebuah namespace yang sama. Misalnya, wordress dapat
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
pada beberapa kasus, di dalam sebuah _namespace_ yang sama. Misalnya, wordpress dapat 
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: wordpress
+    app.kubernetes.io/managed-by: helm
+```
+
+## Aplikasi dan Instans Aplikasi
+
+Sebuah aplikasi dapat diinstal satu atau lebih di dalam kluster Kubernetes dan, 
+pada beberapa kasus, di dalam sebuah namespace yang sama. Misalnya, wordress dapat 
+diinstal lebih dari satu kali dimana website yang berbeda merupakan hasil instalasi yang berbeda.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
diinstal lebih dari satu kali dimana situs web yang berbeda merupakan hasil instalasi yang berbeda.
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 14]
```diff
@@ -0,0 +1,179 @@
...
+haruslah memiliki nama yang unik.
+
+## Contoh
+
+Untuk memberikan ilustrasi dengan cara yang berbeda pada penggunaan label, contoh di bawah ini 
+memiliki tingkat kompleksitas yang cukup beragam.
+
+### Sebuah Aplikasi _Stateless_ Sederhana
+
+Bayangkan sebuah kasus dimana sebuah aplikasi _stateless_ di-deploy
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
Bayangkan sebuah kasus dimana sebuah aplikasi _stateless_ di-_deploy_
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 15]
```diff
@@ -0,0 +1,179 @@
...
+
+## Contoh
+
+Untuk memberikan ilustrasi dengan cara yang berbeda pada penggunaan label, contoh di bawah ini 
+memiliki tingkat kompleksitas yang cukup beragam.
+
+### Sebuah Aplikasi _Stateless_ Sederhana
+
+Bayangkan sebuah kasus dimana sebuah aplikasi _stateless_ di-deploy 
+menggunakan `Deployment` dan `Service`. Di bawah ini merupakan
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]: for API resource, use camelcase without backtick
```suggestion
menggunakan Deployment dan Service. Di bawah ini merupakan 
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 16]
```diff
@@ -0,0 +1,179 @@
...
+memiliki tingkat kompleksitas yang cukup beragam.
+
+### Sebuah Aplikasi _Stateless_ Sederhana
+
+Bayangkan sebuah kasus dimana sebuah aplikasi _stateless_ di-deploy 
+menggunakan `Deployment` dan `Service`. Di bawah ini merupakan 
+contoh kutipan yang merepresentasikan bagaimana 
+label dapat digunakan secara sederhana.
+
+`Deployment` digunakan untuk memastikan Pod dijalankan untuk aplikasi itu sendiri.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
Deployment digunakan untuk memastikan Pod dijalankan untuk aplikasi itu sendiri.
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 17]
```diff
@@ -0,0 +1,179 @@
...
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: myservice
+    app.kubernetes.io/instance: myservice-abcxzy
+...
+```
+
+`Service` digunakan untuk mengekspos aplikasi.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
Service digunakan untuk mengekspos aplikasi.
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 18]
```diff
@@ -0,0 +1,179 @@
...
+    app.kubernetes.io/name: myservice
+    app.kubernetes.io/instance: myservice-abcxzy
+...
+```
+
+### Sebuah Aplikasi Web dengan Basis Data
+
+Bayangkan sebuah aplikasi yang lebih kompleks: sebuah aplikasi web (WordPress)
+yang menggunakan basis data (MySQL), yang diinstal menggunakan Helm. 
+Kutipan berikut merepresentasikan objek yang di-deploy untuk aplikasi ini.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
Kutipan berikut merepresentasikan objek yang di-_deploy_ untuk aplikasi ini.
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 19]
```diff
@@ -0,0 +1,179 @@
...
+
+Berikut merupakan konfigurasi `Deployment` yang digunakan untuk WordPress:
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
Berikut merupakan konfigurasi Deployment yang digunakan untuk WordPress:
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 20]
```diff
@@ -0,0 +1,179 @@
...
+    app.kubernetes.io/name: wordpress
+    app.kubernetes.io/instance: wordpress-abcxzy
+    app.kubernetes.io/version: "4.9.4"
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/component: server
+    app.kubernetes.io/part-of: wordpress
+...
+```
+
+`Service` yang digunakan untuk mengekspos WordPress:
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
Service yang digunakan untuk mengekspos WordPress:
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 21]
```diff
@@ -0,0 +1,179 @@
...
+    app.kubernetes.io/name: wordpress
+    app.kubernetes.io/instance: wordpress-abcxzy
+    app.kubernetes.io/version: "4.9.4"
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/component: server
+    app.kubernetes.io/part-of: wordpress
+...
+```
+
+MySQL diekspos sebagai `StatefulSet` dengan metadata yang digunakan untuk StatefulSets tersebut serta aplikasi yang menggunakannya:
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
MySQL diekspos sebagai StatefulSet dengan metadata yang digunakan untuk StatefulSet tersebut serta aplikasi yang menggunakannya:
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 22]
```diff
@@ -0,0 +1,179 @@
...
+    app.kubernetes.io/name: mysql
+    app.kubernetes.io/instance: mysql-abcxzy
+    app.kubernetes.io/version: "5.7.21"
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: wordpress
+...
+```
+
+`Service` yang digunakan untuk mengekspos MySQL sebagai bagian dari WordPress:
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
Service yang digunakan untuk mengekspos MySQL sebagai bagian dari WordPress:
```

File: content/id/docs/concepts/overview/working-with-objects/common-labels.md [hunk 23]
```diff
@@ -0,0 +1,179 @@
...
+    app.kubernetes.io/name: mysql
+    app.kubernetes.io/instance: mysql-abcxzy
+    app.kubernetes.io/version: "5.7.21"
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: wordpress
+...
+```
+
+Dengan `StatefulSet` MySQL dan `Service` kamu dapat mengetahui informasi yang ada pada MySQL dan Wordpress.
```
Comment (girikuncoro) [inline content/id/docs/concepts/overview/working-with-objects/common-labels.md]:
```suggestion
Dengan StatefulSet MySQL dan Service kamu dapat mengetahui informasi yang ada pada MySQL dan Wordpress.
```

**Comment by girikuncoro** [comment]
@irvifa any update on this?

**Comment by girikuncoro** [comment]
@irvifa this looks good, thanks for your continuous contribution to kubernetes docs and community! 🎉 

/lgtm
/approve

**Pull request merged**